<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fixationTest</class>
 <widget class="QWidget" name="fixationTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2200</width>
    <height>1400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>1080</height>
    </rect>
   </property>
   <widget class="FixationTest" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1920</width>
      <height>1080</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(85, 255, 255)</string>
    </property>
   </widget>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>1970</x>
     <y>30</y>
     <width>180</width>
     <height>60</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent; 
background:#3c3c3c;
color: white;
border-radius:20px;</string>
   </property>
   <item>
    <property name="text">
     <string>        设备</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_start">
   <property name="geometry">
    <rect>
     <x>1970</x>
     <y>160</y>
     <width>180</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent; 
background:#3c3c3c;
color: white;
border-radius:20px;</string>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QLabel" name="displayLabel">
   <property name="geometry">
    <rect>
     <x>1520</x>
     <y>1080</y>
     <width>401</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(212, 255, 255);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>1090</y>
     <width>871</width>
     <height>301</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(220, 255, 247);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_over">
   <property name="geometry">
    <rect>
     <x>1970</x>
     <y>440</y>
     <width>180</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent; 
background:#3c3c3c;
color: white;
border-radius:20px;</string>
   </property>
   <property name="text">
    <string>结束</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_last">
   <property name="geometry">
    <rect>
     <x>1970</x>
     <y>300</y>
     <width>180</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent; 
background:#3c3c3c;
color: white;
border-radius:20px;</string>
   </property>
   <property name="text">
    <string>下一个</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FixationTest</class>
   <extends>QWidget</extends>
   <header location="global">fixationtest.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
